package.json

```diff
@@ -23,15 +23,16 @@
     },
     "homepage": "https://github.com/wikimedia-gadgets/types-mediawiki#readme",
     "dependencies": {
-        "@types/jquery": "^3.5.5"
+        "@types/jquery": "^3.5.5",
+        "@types/oojs-ui": "^0.46.0"
```

Comment:
Would this create a circular dependency issue? I would assume `@types/oojs-ui` itself depends on this package. cc @diskdance @ChlodAlejandro

Reply:
See [this](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/15024fd054cd63fa11739f178a166d7c836b248e/types/oojs-ui/package.json) and [this](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/15024fd054cd63fa11739f178a166d7c836b248e/types/oojs/package.json), they do not depend on this package.

Reply:
Alright I guess let's do this then